language-design.md

```diff
@@ -106,7 +113,9 @@ you do so? If not, why not?
 
 **Response**
 
+These two experiences of natural languages seem extremely at odds with one another. While Pavlus argues that natural language is preferable to the typical syntax in programing languages, Cook has anecdotal evidence that says the contrary. Both experiences seem to agree that natural language is significantly easier to read, but they disagree on the ease of actually writing programs with natural language. However, we found it unclear whether or not the subjects in Cook’s experiment were novice or professional programmers. Had they been professional programmers, it would be more difficult to argue that natural language does not have a place in programming, as we felt natural language would be particularly helpful for those just learning how to program. It makes sense that professional programmers accustomed to the “abstruse” syntax would find AppleScript to be far more troublesome. Additionally, each experiment seemed to differ with respect to the context of the situation: in the experiment cited by Pavlus, it seems the subjects were given no instructions, whereas in Cook’s experiment, the subjects seemed to have some form of guidance. 
 
+We would choose to include natural language, in a very limited scope. Our incorporation of natural language in a DSL would be in very local instances, such as function or method names and the names of built in types or objects, these names are useful to the humans interacting with the code in making it clear what such things do, but when we write something that is a less concrete idea in a natural language, such as a for loop, we would use traditional computer programming idioms.
```

Comment:
I didn't say this in my response because it hadn't occurred to me at the time, but I think when we talk about natural language in programming, we need to separate the words from the syntax. I think you've realized this here, by choosing to give things natural language names, but not using natural language syntax. After all, it's much easier for a function to adhere to the principle of least astonishment if it does exactly what it says on the tin than it is if the function is called "asdf," but a compiler (or interpreter) can't understand slight changes in syntax as meaning the same thing, so best not to make the syntax look like it can be changed slightly without changing the meaning.